🐛(tests) override AI feature flag in config test

ReinforcedKnowledge opened the pull request:
The env.d/development/common file sets
AI_FEATURE_ENABLED=true.
When pytest starts it imports these variables, so
the /api/v1.0/config endpoint returns
AI_FEATURE_ENABLED=True and the test_api_config
assertion fails.

Explicitly overriding AI_FEATURE_ENABLED=False in
test_api_config restores the expected behaviour
and makes the whole test-suite green.

lunika commented:
Hi,

Thank you for your contribution.

Can you sign your commit (see contributing.md file) and rebase your PR to be up to date with the main branch ?

Thanks

ReinforcedKnowledge commented:
Hi!

Thank you for your comment and for accepting my very small contribution.

I just rebased the branch :)

When you say signing the commit, do you mean the simple `git commit --signoff` (also shortened as `-s`) or do you mean the `--gpg-sign`? Because I did read `CONTRIBUTING.md` before and doing `git commit --signoff` just appended this to my commit message: `Signed-off-by: ReinforcedKnowledge <reinforced.knowledge@gmail.com>`.

Let me know if there are better ways for doing this.

lunika commented:
Oh I apologize I thought that the contributing guide was updated with the commit signature but in fact no.

You can see here about what I'm talking about https://github.com/suitenumerique/docs/pull/895/files#diff-eca12c0a30e25b4b46522ebf89465a03ba72a03f540796c979137931d8f92055R7

ReinforcedKnowledge commented:
Thank you again! I just signed the commit properly and pushed it. I don't know if you need a real identity for the commits so I just used my real name instead of my GitHub's username. Is that okay?

lunika commented:
Thanks for signing the commit, the way you did is fine.
The Ci is running, it seems it is failing, at least the linting.

ReinforcedKnowledge commented:
Hi, sorry for my late reply. I just rebased the branch and added a CHANGELOG.md entry following CONTRIBUTING.md's recommendation (and the commit is signed as well).

It is weird that the linting of the CI is failing. I just ran `make lint` and it passes without issues. I pass 100% of the tests with `make test`. I haven't tried `make 

I haven't tried the frontend linting (`make frontend-lint`) since I haven't touched that part at all.

lunika commented:
I merged your commit in the main branch. The commits remains signed off with your email.
Thanks for your contribution
